spec/cloudinary_spec.rb

```diff
@@ -28,5 +28,16 @@
       Cloudinary::config_from_url ENV["CLOUDINARY_URL"]
       expect(Cloudinary::config.foo.bar).to eq 'value'
     end
+    it "should raise an exception if the CLOUDINARY_URL doesn't start with 'cloudinary://'" do
+      invalid_cloudinary_urls = [
+        "",
+        "https://key:secret@test123",
+      ]
+      invalid_cloudinary_urls.each do |cloudinary_url|
+        ENV["CLOUDINARY_URL"]  = cloudinary_url
```

Comment:
Why do you need to assign `cloudinary_url` to `ENV["CLOUDINARY_URL"]` and after that pass it to the function?

Reply:
Good question. That's what I was wondering since the test above this one was doing the same thing so I thought it might've needed to be set as an ENV as well. Removed.